code/lambda_handlers.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import print_function
+import sys
+import log_helper
+sys.path.append('lib')
+import numpy
+from jsonschema import validate
+import validation_json_schemas as schemas
+
+logger = log_helper.getLogger(__name__)
+
+
+def fv_handler(request, context):
+    """
+    Future Value calculation
+    :param request: Dict containing the parameters to pass to the formula.
+    :param context: Lambda execution context
+    :return: Dict with a 'result' entry containing the result of the calculation
+    """
+    schema = schemas.fv_schema
+
+    try:
+        validate(request, schema)
+        logger.info("FV request: {}".format(request))
+        args = [request['rate'], request['nper'], request['pmt'], request.get('pv', 0), request.get('type', 0)]
+        logger.info("Calling numpy.fv with args: {}".format(args))
+        return {'result': numpy.fv(*args)}
+    except Exception as ex:
+        logger.error("Encountered an error while handling FV request with args: {}. Exception: {}".format(request, ex))
+        raise ex
```

Comment:
Agreed - I was focused on validation for this PR and assumed we'd clean up exception handling in the future. Happy to clean it up. 

Regarding logging and raising, I agree in general, though I get in the habit of doing this from Lambda so that any errors are formatted using the log format defined in the logging library. Doesn't matter for this project, but this helps when you have downstream log aggregators (ie Splunk, LogStash, etc) parsing from CWL. Then I raise so that the function error metrics are accurate. Is there a better way to accommodate this? Happy to remove it here given the scope of this project.